# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/ppc-ioctl.h.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Needed for at least popt (sigh), should be reviewed for correctness ...

  - Rene Rebe <rene@exactcode.de>

--- dietlibc-0.29/include/sys/ppc-ioctl.h.vanilla	2006-03-28 10:43:28.087690250 +0200
+++ dietlibc-0.29/include/sys/ppc-ioctl.h	2006-03-28 10:43:32.391959250 +0200
@@ -150,7 +150,6 @@
 #define TIOCMIWAIT	0x545C	/* wait for a change on serial input line(s) */
 #define TIOCGICOUNT	0x545D	/* read serial port inline interrupt counts */
 
-#if 0 /* These are defined this way on Alpha - maybe later. */
 /* Socket-level I/O control calls. */
 
 #define FIOGETOWN	_IOR('f', 123, int)
@@ -161,4 +160,3 @@
 #define SIOCGPGRP	_IOR('s', 9, pid_t)
 
 #define SIOCGSTAMP	0x8906		/* Get stamp - linux-specific */
-#endif
